src/templates/tournaments/tournament_update.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Create/Update Tournament</h1>
```

Comment:
Since we are only updating tournaments at this URL, we should change this to just "Update Tournament".